### ISSUE TITLE
Minecraft Dungeons (1672970)

### ISSUE THREAD

**Issue by karlreinert** [ValveSoftware/Proton#5246]
Game installs nicely through Steam. BUT when I try to run it I get the intor videos, then the backdrop screen after the Unreal Engine screen, with an animated block thing in the right hand lower corner, ambient sound with scary music, then black screen and game crashes.

I've tried using Proton Experimental, Proton 6.3-7 and Proton 6.19-GE-2. All with the same results.

Sysinfo is as follows:
OS: Garuda Linux
KERNEL: 5.14.14-zen1-1-zen
CPU: Intel Core i7-4720HQ @ 2.60GHz
GPU: NVIDIA NVIDIA GeForce GTX 960M
GPU DRIVER: NVIDIA 470.74
RAM: 16 GB

Proton log results
[steam-1672970.log](https://github.com/ValveSoftware/Proton/files/7403726/steam-1672970.log)

**Comment by kisak-valve** [comment]
Hello @karlreinert, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report. Also, please share a Proton log of the game with Proton 6.3 or Experimental.

The detail I'm interested in checking is if the game is trying to run on the Intel GPU or nVidia GPU. You have a Haswell generation Intel GPU, and the video drivers for that generation provide incomplete support for Vulkan. Crashes and/or visual corruption are expected with that generation of Intel GPU.

**Comment by karlreinert** [comment]
Updated Info:

System Info: https://gist.github.com/karlreinert/ab25c4f834c47db0c42219fae089943c

Steam log: [steam-1672970.log](https://github.com/ValveSoftware/Proton/files/7412491/steam-1672970.log)

I've seen that message. Then the question is how do I avoid Vulkan trying to use shaders for Intel card?

**Comment by kisak-valve** [comment]
It would have been nice to also skim over the extended diagnostics information in your system information, but from what I'm seeing, your system is configured to run X on the nVidia GPU, and the Proton log tells us that DXVK is interacting with both the Intel and nVidia GPUs.

You could try setting the game's launch options to `__NV_PRIME_RENDER_OFFLOAD=1 __VK_LAYER_NV_optimus=NVIDIA_only %command%` or try the workaround at https://github.com/ValveSoftware/steam-runtime/issues/312#issuecomment-739441559 and see if either of them help.

**Comment by karlreinert** [comment]
I've been thinking the same, and trying to figure out how to make vulkan not see the intel GPU. Disabeling it if you will. 

I did both the things you wrote. and even combining them. And no luck. Hoping I understood the wolkaround correctly and renaming the .json files and doping nothing else.

I've also been thinking. This happens when the menu normally pops. as the ingame sound starts. I get a few seconds of music. So normally on a windows machine you would have the first few settings popping. So could this be something sound related?

I've tried running it without network enabled. Just in case.

Last thing I can think of is that somehow making it run in window mode would help? Or maybe making it run in directX 9 instead of 11??

**Comment by karlreinert** [comment]
after fiddeling abit with this, I am yet to get any further than the first loading screen. And I do belive that it is the INTEL card in my machine that is making a fuzz. Ironically I can get the game to run without any problems on my "office" machine witch only has an Intel GPU, and far to little resources to play with. And I recon it is the fact that my "gaming" rig is somewhat old and that the support for haskell on the Intel card in my Laptop hasnt been done yet. 

LAst I tried was DXVK_FILTER_DEVICE_NAME="NVIDIA GTX" %command%, and then I got the message that the game needed DX11 and not only Dx10. SO I guess that did something. Will check with thte logs tomorrow.

**Comment by teppyboy** [comment]
I believe this is caused by the nVidia driver, not because of DXVK or Proton because I have a GTX 960M GPU too and when I play the game on Windows it works normally but on Linux (when using nVidia GPU, because lavapipe software rendering works for me) I got the same behaviour like you and the log printed:
```
err: DxvkSubmissionQueue: Failed to sync fence: VK_ERROR_DEVICE_LOST
err: DxvkSubmissionQueue: Command submission failed: VK_ERROR_DEVICE_LOST
```
A temporary workaround that sometimes get through that black screen and go to the login screen for me is installing their beta Vulkan driver (through AUR) and deleting the wineprefix before launching the game.

Btw, when you look up the DXVK error, you'll see it occur in nVidia like [Vk_error_device_lost in many game titles](https://forums.developer.nvidia.com/t/vk-error-device-lost-in-many-game-titles/164513) and [Watchdogs 2: Failed to sync fence: VK_ERROR_DEVICE_LOST](https://github.com/doitsujin/dxvk/issues/2019) and some on Windows lol

tl;dr: vulkan implementation in nvidia driver sucks or my GPU is too outdated for DXVK to work

**Comment by karlreinert** [comment]
I just find it pretty funny that I have less than no issues running Conan Exiles, But Minecraft Dungeons keeps denying me. But alas after some updates on Proton and Nvidia drivers I know get through tha videos and get a black screen for alittle longer before it quits on me. So some things have changed:D

**Comment by coreybruce** [comment]
Game runs well but my sister and I are having issues where the game will randomly freezes/crashes 

---

Minecraft Dungeons Freezing & crashing at random

Issue transferred from https://github.com/ValveSoftware/Proton/issues/5647.
@coreybruce posted on 2022-03-05T15:15:38:

# Compatibility Report
- Name of the game with compatibility issues: Minecraft Dungeons
- Steam AppID of the game: 1672970

## System Information
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/coreybruce/dcd2bec911594c9212d351ac2e7a3f98
- Proton version: 7.0.1

## I confirm:
- [ x] that I haven't found an existing compatibility report for this game.
- [ x] that I have checked whether there are updates for my system available.



Log via gist: https://gist.github.com/coreybruce/3aa6949fc10a42194fbc85d0f55c0131

## Symptoms 
Will freeze at ranmdom, sometimes crash
## Reproduction

Run game using proton 7.0.1

**Comment by kisak-valve** [comment]
Minecraft Dungeons Freezing & crashing at random

Issue transferred from https://github.com/ValveSoftware/Proton/issues/5647.
@coreybruce posted on 2022-03-05T15:15:38:

# Compatibility Report
- Name of the game with compatibility issues: Minecraft Dungeons
- Steam AppID of the game: 1672970

## System Information
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/coreybruce/dcd2bec911594c9212d351ac2e7a3f98
- Proton version: 7.0.1

## I confirm:
- [ x] that I haven't found an existing compatibility report for this game.
- [ x] that I have checked whether there are updates for my system available.



Log via gist: https://gist.github.com/coreybruce/3aa6949fc10a42194fbc85d0f55c0131

## Symptoms 
Will freeze at ranmdom, sometimes crash
## Reproduction

Run game using proton 7.0.1

**Comment by coreybruce** [comment]
> I just find it pretty funny that I have less than no issues running Conan Exiles, But Minecraft Dungeons keeps denying me. But alas after some updates on Proton and Nvidia drivers I know get through tha videos and get a black screen for alittle longer before it quits on me. So some things have changed:D

That's super odd, I am playing the game on a 1060 6gb card but the game will crash/freeze at random

**Comment by coreybruce** [comment]
Can someone please fix this bug, it's really annoying and can ruin the game experience with the freezing and crashing :(

**Comment by LiamDawe** [comment]
Just to confirm, the game randomly freezing is an issue here too. I've seen it on Desktop Linux and now on Steam Deck too, sadly didn't catch a log either time.

Usually happens after a mission is finished, when you get to open a chest. Will try to play more and get a log of when it happens.

**Comment by coreybruce** [comment]
> Just to confirm, the game randomly freezing is an issue here too. I've seen it on Desktop Linux and now on Steam Deck too, sadly didn't catch a log either time.
> 
> Usually happens after a mission is finished, when you get to open a chest. Will try to play more and get a log of when it happens.

Try using proton experimental or better yet proton-ge, it's a more stable on proton experimental or ge and doesn't do that very often and unless you are in a multiplayer game and the host leaves.

**Comment by christofin** [comment]
Hi, I think I have discovered what is prompting Minecraft Dungeons to freeze & crash so much when running on Linux through Proton (at least for me, on the Steam Deck). 

I can replicate this issue on my Steam Deck 100% of the time, and on Windows 0% of the time. 

Simply put, in any online lobby, the game will crash on Linux when one of the other players leaves the lobby (either via main menu or Alt + F4). As soon as any other player leaves, the game crashes on the Steam Deck 100% of the time. 

Can anyone do a quick test to confirm my findings? I've done numerous tests on Windows and Steam Deck, and the player-leaving-crashing issue ONLY happens on the Steam Deck (regardless of whether the Deck is the host or not).

**Comment by coreybruce** [comment]
I don't have a Steam Deck but I can do some tests

**Comment by notabirb1** [comment]
I can launch and play the game fine, but I get a ridiculous amount of lag after opening the inventory. Has anyone else had this issue?

**Comment by nicholaschum** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/5246#issuecomment-1172615289 

I can confirm this issue. I'm running the latest Steam OS 3.3.1 beta (as of posting) and running on ProtonGE 29. 

As soon as anyone leaves, disconnects or crashes on their end, the Steam Deck installation of MCD freezes up.

**Comment by coreybruce** [comment]
Game works great but crashes a lot still, no idea if it matter if you are playing multiplayer or not. I am using Proton Experimental

Here is the latest log:
https://gist.github.com/coreybruce/60df93f76d8cc0bc96313f842aef7616

**Comment by ghost** [comment]
Hello,

maybe not worth to report because i do use FreeBSD with wine-proton 6.3 (which is still fine)

but... I was able to workaround the random crashes by disabling the "winegstreamer" library for Dungeons-Win64-Shipping.exe

**Comment by coreybruce** [comment]
Oh how do you do that? I might have to test to see if that helps

**Comment by ghost** [comment]
1. You open winecfg in your gameprefix
2. Add the exe (Dungeons-Win64-Shipping.exe) to Applications 
3. Then you select the added exe and switch to Libraries and create a new override as winegstreamer (disabled)

You should enter winecfg with protons wine into your gameprefix or things could get mixed up.

AFAIK there should be some helper like protontricks.

You can see if the entry is working by having an black screen at start instead of working Microsoft and unreal engine intro.

**Comment by kisak-valve** [comment]
It's probably easier to try setting the game's launch options to `WINEDLLOVERRIDES="winegstreamer=d" %command%` and see if it has the same effect.

**Comment by AMDBartek** [comment]
I was also having crashing issues in Minecraft Dungeons and I can confirm that using Proton Experimental/Proton-GE and `WINEDLLOVERRIDES="winegstreamer=d"` in my launch options fixed the crashing for me.

It still sometimes crashes but very infrequently now.

Edit: Using an Nvidia GeForce RTX 3060 Laptop GPU but my cousin who has an AMD GPU is also crashing.

**Comment by alectries** [comment]
I was having graphical issues on Steam Deck with Proton Experimental, reverted to Proton 7 and issues stopped.

**Comment by kisak-valve** [comment]
Hello @alectries, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the render snafu, and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.) Also, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

Can you share a couple screenshots of the render issues?

**Comment by alectries** [comment]
As requested,

Sysinfo gist: https://gist.github.com/alectries/9e443cca2f649e527d7defcec0826732

Proton log attached.
[steam-1672970.log](https://github.com/ValveSoftware/Proton/files/10299205/steam-1672970.log)

Photos attached.
![A61B022E1806CD9BA471F2A54948C5B7126C174C](https://user-images.githubusercontent.com/100505370/209451960-2db4779a-6314-4784-999b-cedc80226b47.jpeg)
![1D2B8C3ECFAC905414A6FE72999A0173EB81DADA](https://user-images.githubusercontent.com/100505370/209451962-4c52a8ab-821a-419b-8de2-b1bc8ce1ff74.jpeg)
![C4C1F9C8806CCFEA6E05796B0A1CC17C3D1AABD4](https://user-images.githubusercontent.com/100505370/209451965-b5b3792f-3357-445a-92a4-90ac150017d5.jpeg)
1. Full-screen SMPTE color bars, which occurs in place of the Xbox Game Studios and other branding videos on launch.
2. Color bars appearing behind "You Died" where other textures normally appear.
3. Color bars appearing in place of and behind TNT and arrow icons on hotbar.
I have noticed similar effects strung throughout gameplay. None of these issues affect ability to use the game, but they are annoying.

As previously mentioned, reverting to Proton 7.0-5 completely resolves these issues on Steam Deck.

**Comment by kisak-valve** [comment]
Thanks, it looks like the log you attached is from using the game with Proton 6.3 instead of Proton Experimental. That hints that there's just some video playback functionality which has improved between Proton 6.3 and Proton 7.0.

**Comment by alectries** [comment]
Odd. I was having no issues with the game yesterday, but began having these issues today. Switching to 7.0 solved the issue. I assumed Proton Experimental had received an update which caused the error.

Is there some reason the Deck would have automatically reverted to 6.3? I don't believe the game itself has received an update that caused the issue, but I could be mistaken. Regardless, the status on Deck is currently "unsupported," presumably due to Proton issues, which is inaccurate.

**Comment by notabirb1** [comment]
I'm not sure why, but opening the inventory massively lowers the framerate until I restart the game (on my machine, anyway), is this a known issue?

**Comment by wispii** [comment]
Hello, i've been playing minecraft dungeons on my Steam Deck, but every time I play it the game crashes after a few minutes. Tried changing different versions of proton and using  `WINEDLLOVERRIDES="winegstreamer=d" `, but it only delays the crash rather than fixing it.

**Comment by ChristianJohnsten** [comment]
Like others have said, launch options and other versions of Proton only delays the crash.

I've tried all of the following launch options:
- `gamemoderun %command%`
- `WINEDLLOVERRIDES="winegstreamer=d"`
- `gamemode WINEDLLOVERRIDES="winegstreamer=d" %command%`

And I've tried these Proton versions:
- Proton 6.3-8
- Proton 7.0-5
- Proton Experimental

I heard some people on ProtonDB had better luck with these versions of Proton Glorious Eggroll, but again, these were only delayed the crash or freeze. [https://www.protondb.com/app/1672970](https://www.protondb.com/app/1672970)
- Proton-6.16-GE-1
- GE-Proton-7-26
- GE-Proton-7-33
- GE-Proton-7-43

I dunno if this information will be helpful as well, but I've noticed most crashes happen within 15-30ish minutes. And when it does it's either:
- Completely random
- When you complete the level and get to the level completion stats screen
- Sometimes when TNT explodes

Hoping this info might be useful

**Comment by ghost** [comment]
I've played for around 30/45 mins alone and no problems at all. When I play with a fried of mine it tends to crash every 10/15 mins. The only change I've made is `WINEDLLOVERRIDES="winegstreamer=d"`.

Sysinfo is as follows:
OS: Gentoo Linux
KERNEL: 6.1.8-gentoo-dist
CPU: Intel Core i5 12400F @ 3.5GHz
GPU: NVIDIA GeForce GTX 1650
GPU DRIVER: nvidia-drivers 525.85.05
RAM: 64 GB
Steam version: 1.0.0.74
Proton version: proton-experimental

**Comment by ghost** [comment]
I've run steam with -vv and that's the debug portion when it crash
```
...
Steam_SetMinidumpSteamID:  Caching Steam ID:  76561198094581550 [API loaded no]
CAppInfoCacheReadFromDiskThread took 0 milliseconds to initialize
[0125/162828.347:ERROR:network_change_notifier_win.cc(156)] WSALookupServiceBegin failed with: 8
Installing breakpad exception handler for appid(steam)/version(1671236931)
Installing breakpad exception handler for appid(steam)/version(1671236931)
Installing breakpad exception handler for appid(steam)/version(1671236931)
Installing breakpad exception handler for appid(steam)/version(1671236931)
[2023-01-25 16:30:12] Background update loop checking for update. . .
[2023-01-25 16:30:12] Verifica della disponibilità di aggiornamenti...
[2023-01-25 16:30:12] Downloading manifest: https://client-update.akamai.steamstatic.com/steam_client_ubuntu12
[2023-01-25 16:30:12] Manifest download: send request
[2023-01-25 16:30:12] Manifest download: waiting for download to finish
[2023-01-25 16:30:13] Manifest download: finished
[2023-01-25 16:30:13] Download skipped by HTTP 304 Not Modified
[2023-01-25 16:30:13] Nothing to do

(wine:270207): GStreamer-CRITICAL **: 16:51:04.475: gst_element_set_state: assertion 'GST_IS_ELEMENT (element)' failed

(wine:270207): GStreamer-CRITICAL **: 16:51:04.475: gst_object_unref: assertion 'object != NULL' failed

(wine:270207): GStreamer-CRITICAL **: 16:51:04.475: gst_object_unref: assertion 'object != NULL' failed
free(): invalid pointer
ThreadGetProcessExitCode: no such process 270204
ThreadGetProcessExitCode: no such process 270189
ThreadGetProcessExitCode: no such process 270182
ThreadGetProcessExitCode: no such process 270176
ThreadGetProcessExitCode: no such process 270170
ThreadGetProcessExitCode: no such process 270148
ThreadGetProcessExitCode: no such process 270145
ThreadGetProcessExitCode: no such process 270139
ThreadGetProcessExitCode: no such process 270135
pid 270141 != 270140, skipping destruction (fork without exec?)
Game process removed: AppID 1672970 "/home/scardracs/.local/share/Steam/ubuntu12_32/reaper SteamLaunch AppId=1672970 -- /home/scardracs/.local/share/Steam/ubuntu12_32/steam-launch-wrapper -- '/home/scardracs/.local/share/Steam/steamapps/common/SteamLinuxRuntime_soldier'/_v2-entry-point --verb=waitforexitandrun -- '/home/scardracs/.local/share/Steam/steamapps/common/Proton - Experimental'/proton waitforexitandrun  '/home/scardracs/.local/share/Steam/steamapps/common/MinecraftDungeons/Dungeons.exe'", ProcID 270207 
ThreadGetProcessExitCode: no such process 270207
ThreadGetProcessExitCode: no such process 270157
ThreadGetProcessExitCode: no such process 270040
Game 1672970 created interface STEAMAPPLIST_INTERFACE_VERSION001 / AppList
Game 1672970 created interface STEAMAPPS_INTERFACE_VERSION008 / Apps
Game 1672970 created interface STEAMHTMLSURFACE_INTERFACE_VERSION_003 / HTMLSurface
Game 1672970 created interface STEAMHTTP_INTERFACE_VERSION002 / HTTP
Game 1672970 created interface STEAMINVENTORY_INTERFACE_V001 / Inventory
Game 1672970 created interface STEAMMUSICREMOTE_INTERFACE_VERSION001 / MusicRemote
Game 1672970 created interface STEAMMUSIC_INTERFACE_VERSION001 / Music
Game 1672970 created interface STEAMREMOTESTORAGE_INTERFACE_VERSION014 / RemoteStorage
Game 1672970 created interface STEAMSCREENSHOTS_INTERFACE_VERSION003 / Screenshots
Game 1672970 created interface STEAMUGC_INTERFACE_VERSION009 / UGC
Game 1672970 created interface STEAMUNIFIEDMESSAGES_INTERFACE_VERSION001 / UnifiedMessages
Game 1672970 created interface STEAMUSERSTATS_INTERFACE_VERSION011 / UserStats
Game 1672970 created interface STEAMVIDEO_INTERFACE_V001 / Video
Game 1672970 created interface SteamController005 / Controller
Game 1672970 created interface SteamFriends015 / Friends
Game 1672970 created interface SteamMatchMaking009 / Matchmaking
Game 1672970 created interface SteamMatchMakingServers002 / MatchmakingServers
Game 1672970 created interface SteamNetworking005 / Networking
Game 1672970 created interface SteamUser019 / User
Game 1672970 created interface SteamUtils008 / Utils
Game 1672970 method call count for IClientConfigStore::GetString : 3
Game 1672970 method call count for IClientConfigStore::GetInt : 3
Game 1672970 method call count for IClientNetworking::AllowP2PPacketRelay : 1
Game 1672970 method call count for IClientUserStats::RequestUserStats : 1
Game 1672970 method call count for IClientUserStats::GetAchievementDisplayAttribute : 312
Game 1672970 method call count for IClientUserStats::SetAchievement : 28
Game 1672970 method call count for IClientUserStats::GetAchievement : 104
Game 1672970 method call count for IClientUserStats::StoreStats : 28
Game 1672970 method call count for IClientUserStats::RequestCurrentStats : 1
Game 1672970 method call count for IClientAppManager::IsAppDlcInstalled : 22
Game 1672970 method call count for IClientApps::BGetDLCDataByIndex : 22
Game 1672970 method call count for IClientApps::GetDLCCount : 2
Game 1672970 method call count for IClientMatchmaking::ReleaseGMSQuery : 3
Game 1672970 method call count for IClientMatchmaking::PollGMSQuery : 55
Game 1672970 method call count for IClientMatchmaking::BeginGMSQuery : 3
Game 1672970 method call count for IClientUtils::RecordSteamInterfaceCreation : 44
Game 1672970 method call count for IClientUtils::GetAPICallResult : 1
Game 1672970 method call count for IClientUtils::IsAPICallCompleted : 16
Game 1672970 method call count for IClientUtils::GetAppID : 51
Game 1672970 method call count for IClientFriends::GetFriendRichPresenceKeyByIndex : 8
Game 1672970 method call count for IClientFriends::GetFriendRichPresenceKeyCount : 13
Game 1672970 method call count for IClientFriends::GetFriendRichPresence : 8
Game 1672970 method call count for IClientFriends::ClearRichPresence : 4
Game 1672970 method call count for IClientFriends::SetRichPresence : 8
Game 1672970 method call count for IClientFriends::GetFriendGamePlayed : 5
Game 1672970 method call count for IClientFriends::GetFriendPersonaState : 5
Game 1672970 method call count for IClientFriends::GetFriendRelationship : 9
Game 1672970 method call count for IClientFriends::GetPersonaState : 1
Game 1672970 method call count for IClientUser::BIsSubscribedApp : 1
Game 1672970 method call count for IClientUser::GetAuthSessionTicket : 5
Game 1672970 method call count for IClientUser::GetSteamID : 103
Game 1672970 method call count for IClientUser::BLoggedOn : 4
Uploaded AppInterfaceStats to Steam

(process:270308): GLib-GObject-CRITICAL **: g_object_unref: assertion `G_IS_OBJECT (object)' failed
Sensori di monitoraggio del sistemasrc/steamexe/main.cpp (254) : Assertion Failed: reaping pid: 270308 -- gameoverlayui

src/steamexe/main.cpp (254) : Assertion Failed: reaping pid: 270308 -- gameoverlayui

Installing breakpad exception handler for appid(steam)/version(1671236931)
assert_20230125165118_33.dmp[271382]: Uploading dump (out-of-process)
/tmp/dumps/assert_20230125165118_33.dmp
assert_20230125165118_33.dmp[271382]: Finished uploading minidump (out-of-process): success = yes
assert_20230125165118_33.dmp[271382]: response: CrashID=bp-6f1143d4-36c5-4fbd-8c2d-864822230125
assert_20230125165118_33.dmp[271382]: file ''/tmp/dumps/assert_20230125165118_33.dmp'', upload yes: ''CrashID=bp-6f1143d4-36c5-4fbd-8c2d-864822230125''

```

**Comment by ghost** [comment]
That's the dump file I was able to take [assert_20230125165118_33.tar.gz](https://github.com/ValveSoftware/Proton/files/10501068/assert_20230125165118_33.tar.gz)

**Comment by Rahzadan** [comment]
Having the same crashing every 15/30 mins on Steam Deck. Using Proton-GE 7.49. Plays 100% perfectly before the crash, however.

**Comment by r1sk01** [comment]
Was this ever fixed?
I'm having the same issue with both the latest version of ProtonGE and Proton Experimental alongside a bunch of select Proton versions, on Arch Linux with the `linux/arch` kernel, distro is EndeavourOS

**Comment by antgly** [comment]
Happening to me as well from a clean install.

**Comment by gazsp** [comment]
Me too.

**Comment by Cederick** [comment]
Sorry, can happily report that the game runs fine on Steam Deck using the latest version of Proton (Proton 9) and I had **zero crashes, stuttering, lags, glitches or anything else in 11 hours of uninterrupted playtime.**

The game shows controller glyphs, the basic configuration runs well and the UI and HUD are readable without issues. That's no big suprise, because it also got ported to Nintendo Switch.

The game is also rated **[platin](https://www.protondb.com/app/1672970)** on ProtonDB.

So, I would request for this game to know only get the **playable** status but instead gets marked as **verified**.

ATTENTION: Steam does choose Proton 6 as default for some reason so you'll have to manually change it to Proton 9.

**Comment by Jaskowicz1** [comment]
Update on this, latest bleeding edge beta for Proton Experimental seems to have broken some of the effects in Minecraft Dungeons.

![image](https://github.com/ValveSoftware/Proton/assets/46899449/db7611ea-fd10-4032-883e-beb22b79bcd4)

This happens with Enchants, The "You Died" screen, and any weapon icon that has a glow (more notably when selecting an award after The Tower).

**Comment by kisak-valve** [comment]
Hello @Jaskowicz1, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the regression and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.) Also, please copy the contents of Steam Runtime Diagnostics from Steam (`Steam` -> `Help` -> `Steam Runtime Diagnostics`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

Please clarify what's the newest version of Proton you know is not affected? Proton 9.0-2? Proton Experimental (non-beta branch + build date)?

**Comment by Jaskowicz1** [comment]
@kisak-valve Hi there, here's my proton log:
[steam-1672970.log](https://github.com/user-attachments/files/16134011/steam-1672970.log)

and here's my steam diagnostics: https://gist.github.com/Jaskowicz1/0484bae58881f5af745d390cad1d834a

For the proton version, the latest that works perfectly is latest Proton Experimental (Build ID: 14944099 / experimental-9.0-20240705).

**Comment by veygax** [comment]
@kisak-valve this now works perfectly on steam deck, can we get this updated to Deck Verified? saves having to go to protondb.
